packages/quick_actions/quick_actions_ios/ios/Classes/QuickActionsPlugin.swift

```diff
@@ -75,14 +66,38 @@ public final class QuickActionsPlugin: NSObject, FlutterPlugin, IOSQuickActionsA
 
   public func applicationDidBecomeActive(_ application: UIApplication) {
     if let shortcutType = launchingShortcutType {
-      handleShortcut(shortcutType)
+      self.handleShortcut(shortcutType)
       self.launchingShortcutType = nil
     }
   }
 
   func handleShortcut(_ shortcut: String) {
-    flutterApi.launchAction(action: shortcut) { _ in
+    self.flutterApi.launchAction(action: shortcut) { _ in
       // noop
     }
   }
+
+  static func parseShortcutItems(_ items: [ShortcutItemMessage]) -> [UIApplicationShortcutItem] {
```

Comment:
this doesn't seem to belong to a static member of `QuickActionsPlugin`. I suggest 2 alternatives - 

1. make them private non-static members of `QuickActionsPlugin` (so they become implementation details `QuickActionsPlugin`). 

2. A more swifty way I would do is to create extension of either `UIApplicationShortcutItem` or `ShortcutItemMessage`.

Reply:
Sounds good, thanks for the feedback!

Reply:
Oh also avoid the naming "parse" since it's not parsing anymore. Something like: 

```
extension ShortcutItemMessage {
  var item: UIApplicationShortcutItem {
    return ...
  }
}
```

Reply:
I don't think we need the tests that I was using this for any longer, since this is now an implementation detail and my understanding is that the swifty thing to do is consider the converter tested, since it is used in code that is already tested.

Reply:
Yes if it's implementation detail, it can only be tested via non-private members. You can delete those tests if it's already covered elsewhere